libs/execution_context/ebpf_maps.c

```diff
@@ -507,8 +536,9 @@ _create_per_cpu_hash_map(_In_ const ebpf_map_definition_t* map_definition)
     if (retval != EBPF_SUCCESS) {
         if (per_cpu_data) {
             for (index = 0; index < per_cpu_data->count; index++) {
-                ebpf_hash_table_t** tables = (ebpf_hash_table_t**)&per_cpu_data->data;
-                ebpf_hash_table_destroy(tables[index]);
+                ebpf_core_per_cpu_hash_table_t* tables = (ebpf_core_per_cpu_hash_table_t*)&per_cpu_data->data;
```

Comment:
if this is common code repeated multiple times, consider making a inline function or macro.

Reply:
Not really common code.